.pre-commit-config.yaml

```diff
@@ -0,0 +1,84 @@
+---
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+ci:
+  skip:
+    - prettier-django
+    # Because these are local hooks it seems like they won't easily run in pre-commit CI
+    - eslint
+    - scss-lint
```

Comment:
Not problematic necessarily, but this config has some extra hooks. E.g., `scss-lint` hook doesn't exist here

Reply:
Thanks, I thought I'd caught all of those.